### PR TITLE
Fix compareObjectTypeWithValueType Method

### PR DESCRIPTION
**Pull request by mdelez**
closes https://dasch.myjetbrains.com/youtrack/issue/DSP-630

### CODE REVIEW

File: projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts [hunk 1]
```diff
@@ -46,4 +46,43 @@ describe('ValueTypeService', () => {
...
         });
     });
+
+    describe('compareObjectTypeWithValueType', () => {
+
+        it('should successfully compare "http://api.knora.org/ontology/knora-api/v2#TextValue" with "ReadTextValueAsString"', () => {
+            expect(service.compareObjectTypeWithValueType(
+                'ReadTextValueAsString',
+                'http://api.knora.org/ontology/knora-api/v2#TextValue'))
+                .toBeTruthy();
```
Comment (tobiasschweizer) [inline projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts:56]: Could you maybe add another (failed) comparison between a text value and a non text type?

**Review by tobiasschweizer — APPROVED** [review]

File: projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts (hunk 1 above)
Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts:56]: text value meaning `http://api.knora.org/ontology/knora-api/v2#TextValue` or `ReadTextValueAsString`?

Comment (tobiasschweizer) [reply projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts:56]: compare a `ReadTextValueAsString` to an IntValue for example. Sorry for being unprecise

Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts:56]: no worries, added in 691a5a8 :)

Comment (tobiasschweizer) [reply projects/dsp-ui/src/lib/viewer/services/value-type.service.spec.ts:56]: thx, good to go then!

**Pull request merged**